submissions/lisnyak-o/popup-kottans/index.html

```diff
@@ -0,0 +1,137 @@
+              </a>
+              <a href="#" class="scroll__item">
+                <img class="scroll__item-img" src="images/contacts.png" alt="contacts">
+                <span class="scroll__item-text">Contacts</span>
+              </a>
+              <a href="#" class="scroll__item">
+                <img class="scroll__item-img" src="images/documents.png" alt="documents">
+                <span class="scroll__item-text">Documents</span>
+              </a>
+              <a href="#" class="scroll__item">
+                <img class="scroll__item-img" src="images/files.png" alt="files">
+                <span class="scroll__item-text">Files</span>
+              </a>
+              <a href="#" class="scroll__item">
+                <img class="scroll__item-img" src="images/mail.png" alt="mail">
+                <span class="scroll__item-text">Mail</span>
+              </a>
+              <a href="#" class="scroll__item">
+                <img class="scroll__item-img" src="images/maps.png" alt="maps">
+                <span class="scroll__item-text">Maps</span>
+              </a>
+              <a href="#" class="scroll__item">
+                <img class="scroll__item-img" src="images/mars.png" alt="mars">
+                <span class="scroll__item-text">Mars</span>
+              </a>
+            </div>
+            <label class="more-btn" for="more-checkbox">More</label>
+            <input type="checkbox" class="popup-btn" id="more-checkbox">
+            <div class="more__items content">
+              <a href="#" class="more__item">
+                <img class="more__item-img" src="images/photos.png" alt="photos">
+                <span class="more__item-text">Photos</span>
+              </a>
+              <a href="#" class="more__item">
+                <img class="more__item-img" src="images/news.png" alt="news">
+                <span class="more__item-text">News</span>
+              </a>
+              <a href="#" class="more__item"><img class="more__item-img" src="images/promo.png" alt="promo">
+                <span class="more__item-text">Promo</span>
+              </a>
+              <a href="#" class="more__item">
+                <img class="more__item-img" src="images/search.png" alt="search">
+                <span class="more__item-text">Search</span>
+              </a>
+              <a href="#" class="more__item">
+                <img class="more__item-img" src="images/translation.png" alt="translation">
+                <span class="more__item-text">Translation</span>
+              </a>
+              <a href="#" class="more__item">
+                <img class="more__item-img" src="images/videos.png" alt="videos">
+                <span class="more__item-text">Videos</span>
+              </a>
+              <a class="kottans" href="https://kottans.org/">Even more from Kottans</a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </header>
+</body>
+
+</html>
```

Comment:
Add an empty new line.
You should always leave a blank line at the end of the file.